### ISSUE TITLE
How to test the model that is trained using our devices?

### ISSUE THREAD

**Issue by CZhangIR** [liux520/EIMN#2]
How do I evaluate the model trained on my datasets? I would be grateful if you could reply as soon as possible.

**Comment by liux520** [comment]
If you want to test the model on your own dataset, the simplest test code is provided in the Demo/test_on_custom_datset.py.
Please feel free to ask if you need further assistance.

**Comment by chunzhang07** [comment]
According to the code you gave, we retrained the EIMN_L_x2 model. In fact, the result obtained was 0.1dB lower than the PSNR of the pre-trained model. Is it related to some parameters, such as seed = 123?

**Comment by liux520** [comment]
In the first stage, we train 1000 epochs with a learning rate of 5e-4 and cos decay. Then we load the best weight from the former phase each time and continue training with the same learning rate. Multiple cycles are performed until this stage converges. (Approximately 38.18 reached, Y channel, crop_border=2)
The second stage trains with a learning rate of 1e-4 and cos decay.  Again, multiple phases until this phase converges. (Reached approximately 38.23, Y-channel, crop_border=2)
The third stage trains with a learning rate of 5e-5 and cos decay. Again, multiple phases until this stage converges. (Reached approximately 38.26, Y-channel, crop_border=2)

**Comment by chunzhang07** [comment]
Thank you very much for your patient answer. I will try again according to the training strategy you mentioned.
